tests/integration/aps/test_utils.py

```diff
@@ -41,20 +46,19 @@ def test_aps_fetch_api(dag: DAG):
     articles_metadata = str.encode(
         json.dumps(aps_api_client.get_articles_metadata(parameters))
     )
-    save_file_in_s3(articles_metadata, repo)
-    assert len(repo.find_all()) == 1
+    save_file_in_s3(articles_metadata, aps_empty_repo)
+    assert len(aps_empty_repo.find_all()) == 1
 
 
 @pytest.mark.vcr
-def test_dag_run(dag: DAG):
-    repo = APSRepository()
-    repo.delete_all()
-    assert len(repo.find_all()) == 0
+def test_dag_run(dag: DAG, aps_empty_repo):
+    assert len(aps_empty_repo.find_all()) == 0
     dag.clear()
     dag.test(
         run_conf={
-            "from_date": "2022-02-05",
-            "until_date": "2022-03-05",
+            "from_date": "2022-02-07",
+            "until_date": "2022-02-07",
         }
     )
-    assert len(repo.find_all()) == 1
+    time.sleep(5)
```

Comment:
the tests on github actions are unstable. You can see that fetch files from the FTP server were not saved before the  `assert len(aps_empty_repo.find_all()) == 1` was checked. So, the file was not found in our s3 pod.
https://github.com/cern-sis/workflows/actions/runs/5679947342/job/15393077928

This test was always passing before, however, now we see that it's crashing. I put `sleep` in order to avoid this issue. But it's not a good solution. Any suggestions?